src/main/java/com/example/backend/RestController/RoomController.java

```diff
@@ -85,4 +87,16 @@ public Message updateReservation(@PathVariable long id) {
          repo.deleteById(id);
          return new Message("room has been deleted");
      }
-}
+     
+     @GetMapping("/fetch")
+     public List<Room> fetchRoom(@RequestBody Room room) {
```

Comment:
What does this method do? Why we need this method? Why the url is /fetch?

Reply:
Fetch is for fetching room data based on a partial match. I fixed the URL in the latest commit.